q2_assembly/quast/tests/test_quast.py

```diff
@@ -12,13 +12,18 @@
 import shutil
 import tempfile
 import unittest
+import warnings
 from filecmp import dircmp
 from subprocess import CalledProcessError
 from unittest.mock import ANY, MagicMock, Mock, call, mock_open, patch
 from zipfile import ZipFile
 
 import pandas as pd
+from future.moves import itertools
```

Comment:
Why are you using itertools from `future.moves`?

Reply:
Actually this was the only option that offered me the combination of parameters I wanted. I have simplified the parameterization in the new commit and I am removing the import.

Reply:
The point is not to remove it. My question is why is the import from `future.moves`? Was it PyCharm's suggestion or ChatGPT's? You should use `product` from `itertools` directly - future.moves is a library that was introduced long time ago for compatibility between Python 2 and 3 and really has no place in our Python 3 code anymore. Or did you try the new method and it didn't work?

Reply:
It was Pycharm's suggestion. It gave me two options and this was the first one. I tried the `product` one before committing and it did not have the `repeat` argument. Actually, for the 3 tuples that I needed for that cases, I believe it was a bit overly complicated😅, that's why I manually wrote them. You can check and let me know what you think.

Reply:
But of course there is a `repeat` argument there - it's the same function... https://docs.python.org/3/library/itertools.html#itertools.product

You're right, though - manual seems a bit simpler 😅

Reply:
Ah never mind, my bad, the second trial was from itertools from `tqdm`, somehow Pycharm does not suggest me the simple built-in itertools...🤔

So I am leaving it manual (it seems cleaner as well😄)